packages/gatsby-source-wordpress/src/steps/source-nodes/create-nodes/process-node.js

```diff
@@ -602,36 +602,21 @@ const replaceNodeHtmlImages = async ({
         const gatsbyTransformerSharpSupportsThisFileType =
           supportedExtensions[fileNode?.extension]
 
-        let fluidResult = null
+        let sharpImageData = null
 
         if (gatsbyTransformerSharpSupportsThisFileType) {
           try {
-            fluidResult = await fluid({
+            sharpImageData = await generateImageData({
               file: fileNode,
               args: {
-                maxWidth,
+                width: maxWidth,
                 quality,
-                pathPrefix,
+                ...pluginOptions?.html?.gatsbyImageOptions,
```

Comment:
```suggestion
                ...(pluginOptions?.html?.gatsbyImageOptions || {}),
```

I think if this isn't an object it'll throw

Reply:
Perhaps we should add default {} to pluginOptions above so it's always set

Reply:
@wardpeet it can be set with a default but a user could make pluginOptions.html false or null manually I believe

Reply:
Actually the user plugin options processors can add a default option https://github.com/gatsbyjs/gatsby/blob/87b70010409df5ff5f199e2a2ac37b15e77cba4f/packages/gatsby-source-wordpress/src/steps/process-and-validate-plugin-options.ts#L74

Reply:
You can add it here more easily https://github.com/gatsbyjs/gatsby/blob/87b70010409df5ff5f199e2a2ac37b15e77cba4f/packages/gatsby-source-wordpress/src/models/gatsby-api.ts#L174 but the user can change it I think

Reply:
Just added a default. I think it's pretty edge casey for a user to add html: null and if they do the build will probably fail immediately and they'll undo that change. So it should be fine :)